source/_components/sensor.zabbix.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Zabbix Sensor"
+description: "Instructions how to integrate Zabbix Triggers sensors within Home Assistant."
+date: 2016-12-13 22:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
```

Comment:
Logo value would preferably be filled out and a logo added to this pull request.